scripts/install_extended.sh

```diff
@@ -125,29 +130,61 @@ case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
     ;;
 esac
 
-tag=$(githubLatestTag hazelcast/hazelcast-commandline-client)
-finalUrl=$(printf "https://github.com/hazelcast/hazelcast-commandline-client/releases/download/v%s/hazelcast-commandline-client_%s_%s.tar.gz" "$tag" "$tag" "$bin_id")
+tag=$(ghExtractTag hazelcast/hazelcast-commandline-client)
+releaseUrl=$(printf "https://github.com/hazelcast/hazelcast-commandline-client/releases/download/v%s/hz-cli_%s_%s" "$tag" "$tag" "$bin_id")
 
-curl -L "$finalUrl" > "hz-cli"
+curl -L --silent "$releaseUrl" --output "hz-cli"
+chmod +x "./hz-cli"
 
 case "$(printf "${SHELL##*bin\/}")" in
     "zsh")
+        if [ ! -d "$HOME/.local" ]; then
+            mkdir $HOME/.local
+        fi
+        if [ ! -d "$HOME/.local/bin" ]; then
+            mkdir $HOME/.local/bin
+        fi
```

Comment:
You can skip existence checks if you create the bin directory using `mkdir -p $HOME/.local/bin`

Reply:
Done!